hls4ml/model/layers.py

```diff
@@ -863,10 +864,9 @@ class SimpleRNN(Layer):
     def initialize(self):
         if self.attributes['return_sequences']:
             shape = [self.attributes['n_timesteps'], self.attributes['n_out']]
-            dims = ['N_TIME_STEPS_{}'.format(self.index), 'N_OUT_{}'.format(self.index)]
         else:
-            shape = [self.attributes['n_out']]
-            dims = ['N_OUT_{}'.format(self.index)]
+            shape = [1, self.attributes['n_out']]
```

Comment:
Is there any way to handle this differently? If we always return sequence(s), that means any HLS/C++ implementation will have to be done like that, which is counter-intuitive.